docs(rocprofiler-compute): document ROCm Optiq analysis workflow (#9457)
ROCm/rocm-systems: projects/rocprofiler-compute/docs/how-to/analyze/optiq.rst

.. meta::
   :description: How to analyze ROCm Compute Profiler data with ROCm Optiq
   :keywords: ROCm Compute Profiler, ROCm Optiq, ROCm, profiler, tool, Instinct,
              accelerator, AMD, analysis, analysis database, SQLite

.. _analyze-with-rocm-optiq:

***********************
Analyze with ROCm Optiq
***********************

.. note::

   The ROCm Compute Profiler integration with ROCm Optiq is in beta.

ROCm Optiq is a graphical application for interactively visualizing ROCm Compute
Profiler analysis data.

> The two tools integrate through the analysis database. Generate one with the ``db``

Reviewer: ```suggestion
ROCm Compute Profiler and ROCm Optiq work together through an analysis database. First, generate an analysis database using the ``db``
```

> :ref:`analysis output format <analysis-output-format>`, then open it in ROCm Optiq.

Reviewer: ```suggestion
:ref:`analysis output format <analysis-output-format>`. Then, open the database in ROCm Optiq.
```

> See :ref:`analysis database schema <analysis-database>` for what it contains.

Reviewer: ```suggestion
For more information about the database contents, see the :ref:`analysis database schema <analysis-database>`.
```

> Before using this workflow, review the `ROCm Optiq system requirements`_ and confirm that

Reviewer: ```suggestion
Before you begin, review the `ROCm Optiq system requirements`_. Also, make sure that
```

> your ROCm Optiq version supports the analysis database schema emitted by your installed

Reviewer: ```suggestion
your version of ROCm Optiq supports the analysis database schema generated by your installed version of
```

> ROCm Compute Profiler version. Version and platform compatibility requirements can change

Reviewer: ```suggestion
ROCm Compute Profiler. Compatibility requirements for ROCm Optiq and ROCm Compute Profiler can change
```

independently.

> See the `ROCm Optiq analysis guide`_ for the current opening workflow and interface

Reviewer: ```suggestion
For information about opening the analysis databases and using the interface, see the `ROCm Optiq analysis guide`_.
```

> details.

Reviewer: ```suggestion
```
